term_proposal/t0037799/proposal.md

```diff
@@ -19,18 +19,30 @@ Quantized models from pytorch would be export to onnx format.
 
 So, I only need to implement 2 things which is the frontend and backend of my inference engine.
 
+Also, I'll provide a python interface for ease of use.
+
 ### 1. The interpreter for onnx operators
 onnx define a lot of operators for neural network model such as conv and maxpool.
 I need to parse the data, according to the parameters to build the corresponding model and pass it to my backend system.
 ### 2. The computing runtime.
 In the computing stage, the runtime need to manage memory, data flow and multi-threading.
 It should maximize the use of SIMD to speed up, and minimize the memory footprint.
 
----
+## Implementation
+
+The prototype will at least support mnist dataset on AlexNet.

```

Comment:
It will be even better if you mention how you'd like to code it.  It will give yourself some time to think about the implementation plan.

Things usually change a lot once you start to code, to the degree that it may be totally different from what you originally think.  A plan doesn't prevent the change, but gets you better prepared for it.